README.md

```diff
@@ -4,6 +4,50 @@ Tool to provide a filterable registry of glTF community projects.
 
 http://github.khronos.org/glTF-Project-Explorer/
 
+
+## Contributions
+
+In order to add your project to the project list, you can fork this 
+repository, add your project information to the
+[`glTF-projects-data.json`](public/data/glTF-projects-data.json) file,
+and submit a pull request with the changes. Alternatively, you can 
+also open an issue that contains the relevant project information
+as described below.
+ 
+The JSON properties that can be contained in the project information 
+are summarized in the following table:
+
+| Property | Type | Description |
+|------|--------|-------------|
+| `name` | `string` | The name of the project. This will be the title of the project card, and turned into a link, leading to the URL that is given as the `link` property |
+| `link` | `string` (URL) | The URL of the homepage of the project.  |
+ | `description` | `string` | A short description of the project. This string can contain basic [Markdown](https://en.wikipedia.org/wiki/Markdown) formatting. |
```

Comment:
The parser we're using supports GitHub flavored Markdown, so the [documentation Github provides](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax) may be more useful than the Wikipedia page.